ee/api/test/test_authentication.py

```diff
@@ -709,8 +709,8 @@ def test_xmlsec_and_lxml(self):
         import xmlsec
         import lxml
 
-        assert "1.3.13" == xmlsec.__version__
-        assert "4.9.4" == lxml.__version__
+        assert "1.3.14" == xmlsec.__version__
+        assert "5.2.1" == lxml.__version__
```

Comment:
Well, I assume this test could have been removed now that we fixed it?

Reply:
I'd opt for keeping it. This seems something we need to manually test with care, so a cowboy updating requirements might accidentally break things.

Reply:
makes sense :)